components/ModEditFacilityForm.vue

```diff
@@ -1,5 +1,208 @@
 <template>
-    <div>
-        hi
+    <div
+        class="submission-form-section"
+    >
+        <h1
+            class="mb-3.5 text-start text-primary-text text-3xl font-bold font-sans leading-normal"
+        >
+            {{ $t('modSubmissionForm.facilityHeading') }}
+        </h1>
+        <span class="mb-3.5 text-center text-primary-text text-2xl font-bold font-sans leading-normal">
+            {{ $t('modSubmissionForm.contactInformation') }}
+        </span>
+        <ModInputField
+            data-testid="submission-form-nameEn"
```

Comment:
I wonder if we should name the test IDs and localization variables something more component specific. Something along the lines of `mod-edit-facility-form-nameEn`. What do you think? Are the localization strings being reused elsewhere?

Reply:
We were thinking that. But this component should be reused and similar to the whole edit submission form. So weren't sure how to go about it

Reply:
It's fine that elements are being reused. However, IDs should always be unique. Otherwise, it's hard to guarantee that you're actually testing what you think you are.

Reply:
Since this is a component that’ll be reused, should we shorten the id to mod-edit-form-etc? As far as I understood, we’ll use this component for submission forms, facility forms, healthcare prof. forms and so on eventually? Or is that not the case?

Reply:
So @theyokohamalife we can conditionally render the `id` based on the screen? Is that the best way?

Reply:
@Ntlja @NabbeunNabi The name of the component implies that it is for editing facilities. Facilities, healthcare professionals, and submissions are separate entities. If this is meant to be raised for something other than facilities, the component name should be changed.

Reply:
@theyokohamalife this is meant to be used to edit the facilities portion of an existing facility or a submission facility. This is part of the refactoring of the submission form for being way too long. This was talked about in the long standup 2 weeks ago. That we want to reuse it since the data style is the same for both. And it will make the file much smaller and can be used for the submission or the existing facility

Reply:
So we can rename the fields `facilityFormFields` but it is so that we can be DRY and not have two duplicate forms for editing and existing facility and one that was submitted since the data is the exact same in terms of what is necessary for our DB

Reply:
If that's the case, the IDs should probably be more component specific as mentioned before. Something like the shorter name @Ntlja  suggested. `mod-edit-facility-nameEn` for example.